main.nf

```diff
@@ -445,6 +397,56 @@ def markdup_versions(task) {
 	"""
 }
 
+process copy_bam {
+
+	tag "$id"
+	cpus 1
+	memory '2GB'
+	time '1h'
+
+	input:
+		set group, id, file(bam), file(bai) from copy_bam_ch
+	
+	output:
+		set group, id, file("${id}_dedup.bam"), file("${id}_dedup.bam.bai") into expansionhunter_bam_choice, dnascope_bam_choice, bampath_start, cov_bam_choice, bam_manta_choice, bam_tiddit_choice, bam_mito_choice, bam_SMN_choice, bam_freebayes_choice, bam_mantapanel_choice, bam_cnvkitpanel_choice, bam_dellypanel_choice, bam_melt_choice, bam_qc_choice, dedup_dummy_choice, bam_bqsr_choice, bam_gatk_choice, verifybamid2_bam_choice
+	script:
+		"""
+		ionice -c 2 -n 7 cp ${bam} "${id}_dedup.copy.bam"
+		ionice -c 2 -n 7 cp ${bai} "${id}_dedup.copy.bam.bai"
+		"""
+
+	stub:
+		"""
+		touch "${id}_dedup.copy.bam"
+		touch "${id}_dedup.copy.bam.bai"
+		"""
+}
+
+// These processes expects ID, group instead of group, ID
+// FIXME: We should really fix this to be part of the channels
+def remap_bam_choice_tuple = { channel ->
+		channel.map { tup -> return tuple(tup.get(1), tup.get(0), tup.get(2), tup.get(3))
+	}
+}
+remap_bam_choice_tuple(cov_bam_choice).set { cov_bam_choice }
```

Comment:
I looked through all the BAM-start channels. These were the ones which remapped id <-> group.